culinary-canvas/capstone-project/frontend/my_capstone_project/src/App.jsx

```diff
@@ -1,24 +1,26 @@
-import React from 'react'
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
-import './App.css'
-import HomePage from './HomePage'
-import Sign_Up_Page from './SignUpPage'
-import Log_In_Page from './LogInPage'
-import UserProfile from './UserProfile'
-import {AuthorizationContext} from './RedirectToAuthentication'
+import React from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import "./App.css";
+import HomePage from "./HomePage";
+import Sign_Up_Page from "./SignUpPage";
+import Log_In_Page from "./LogInPage";
+import UserProfile from "./UserProfile";
+import { AuthorizationContext } from "./RedirectToAuthentication";
+import FindRecipes from "./FindRecipes";
 function App() {
   return (
-     <AuthorizationContext>
-        <Router>
-            <Routes>
-              <Route path="/" element={<HomePage />}/>
-              <Route path='/login' element={<Log_In_Page/>}/>
-              <Route path='/create' element={<Sign_Up_Page />}/>
-              <Route path='/login/:id' element={<UserProfile/>}/>
-            </Routes>
-        </Router>
-      </AuthorizationContext>
-  )
+    <AuthorizationContext>
+      <Router>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/login" element={<Log_In_Page />} />
+          <Route path="/create" element={<Sign_Up_Page />} />
```

Comment:
nit: Can we change these variables to have a consistent naming convention as the ones above/below?
e.g.
`HomePage`, `UserProfile` is not consistent with `Log_In_Page`, this can just be `LogInPage`